src/BIAnalyticsService/CreateDataAssets/CreateDataAssets.cs

```diff
@@ -0,0 +1,174 @@
+            _logger.LogError("episodeId is null or empty.");
+            return req.CreateResponse(HttpStatusCode.BadRequest);
+        }
+
+        var baseUrl = Environment.GetEnvironmentVariable("GetEpisodeUrl");
+        var getEpisodeUrl = $"{baseUrl}?EpisodeId={episodeId}";
+        _logger.LogInformation("Requesting episode URL: {Url}", getEpisodeUrl);
+
+        Episode episode;
+
+        try
+        {
+            var response = await _httpRequestService.SendGet(getEpisodeUrl);
+
+            if (!response.IsSuccessStatusCode)
+            {
+                _logger.LogError($"Failed to retrieve episode with Episode ID {episodeId}. Status Code: {response.StatusCode}");
+                return req.CreateResponse(response.StatusCode);
+            }
+
+            string episodeJson;
+            episodeJson = await response.Content.ReadAsStringAsync();
+            _logger.LogInformation("Episode data retrieved");
+            episode = JsonSerializer.Deserialize<Episode>(episodeJson);
+        }
+
+        catch (Exception ex)
+        {
+            _logger.LogError("Issue when getting episode from {getEpisodeUrl}. \nException: {ex}", getEpisodeUrl ,ex);
+            return req.CreateResponse(HttpStatusCode.InternalServerError);
+        }
+
+        string nhsNumber = "1111111112";
+
+        var baseParticipantUrl = Environment.GetEnvironmentVariable("GetParticipantUrl");
+        var participantUrl = $"{baseParticipantUrl}?nhs_number={nhsNumber}";
+        _logger.LogInformation("Requesting participant URL: {Url}",participantUrl);
+
+        Participant participant;
+
+        try
+        {
+            var participantResponse = await _httpRequestService.SendGet(participantUrl);
+
+            if (!participantResponse.IsSuccessStatusCode)
+            {
+                _logger.LogError($"Failed to retrieve participant data with NHS number {nhsNumber}. Status Code: {participantResponse.StatusCode}");
+                return req.CreateResponse(participantResponse.StatusCode);
+            }
+
+            var participantJson = await participantResponse.Content.ReadAsStringAsync();
+            _logger.LogInformation("Participant data retrieved");
+
+            participant = JsonSerializer.Deserialize<Participant>(participantJson);
+        }
+
+        catch (Exception ex)
+        {
+            _logger.LogError("Issue when getting participant from {participantUrl}. \nException: {ex}", participantUrl, ex);
+            return req.CreateResponse(HttpStatusCode.InternalServerError);
+        }
+
+        var screeningEpisodeUrl = Environment.GetEnvironmentVariable("CreateParticipantScreeningEpisodeUrl");
+        var screeningProfileUrl = Environment.GetEnvironmentVariable("CreateParticipantScreeningProfileUrl");
+
+        try
+        {
+            await SendToCreateParticipantScreeningEpisodeAsync(episode, screeningEpisodeUrl);
+            await SendToCreateParticipantScreeningProfileAsync(participant, screeningProfileUrl);
```

Comment:
We don't need to pass the URLs to these methods as arguments, the methods can access the environment variables